### PR TITLE
fix nativeEngine shadow effect not display for v3.1

### PR DESCRIPTION
**Pull request by troublemaker52025**

### CODE REVIEW

File: cocos/renderer/pipeline/shadow/ShadowFlow.cpp [hunk 1]
```diff
@@ -146,18 +143,32 @@ void ShadowFlow::resizeShadowMap(const Light *light, const uint width, const uin
...
+        colorAttachment.storeOp     = gfx::StoreOp::STORE;
+        colorAttachment.sampleCount = gfx::SampleCount::X1;
+
+        gfx::DepthStencilAttachment depthStencilAttachment;
+        depthStencilAttachment.format         = device->getDepthStencilFormat();
+        depthStencilAttachment.depthLoadOp    = gfx::LoadOp::CLEAR;
+        depthStencilAttachment.depthStoreOp   = gfx::StoreOp::DISCARD;
+        depthStencilAttachment.stencilLoadOp  = gfx::LoadOp::CLEAR;
+        depthStencilAttachment.stencilStoreOp = gfx::StoreOp::DISCARD;
+        depthStencilAttachment.sampleCount    = gfx::SampleCount::X1;
```
Comment (minggo) [inline cocos/renderer/pipeline/shadow/ShadowFlow.cpp:166]: Can initialize struct as
```c++
gfx::ColorAttachment colorAttachment{...};
```

**Comment by minggo** [comment]
Why it has problem in v3.1?

File: cocos/renderer/pipeline/shadow/ShadowFlow.cpp (hunk 1 above)
Comment (troublemaker52025) [reply cocos/renderer/pipeline/shadow/ShadowFlow.cpp:166]: sure

**Comment by troublemaker52025** [comment]
> 
> 
> Why it has problem in v3.1?

Three weeks ago, the v3.1 branch merged many features of the v3.0.1 branch, and some of the new writes were mistakenly discarded during the merge, resulting in the current unusual effect

**Comment by minggo** [comment]
Got it, thanks.

**Pull request merged**